docs/learn-identity.md

```diff
@@ -39,16 +75,48 @@ When a registrar provides judgement, they can select up to six levels of confide
 - Low Quality: The information is low quality or imprecise, but can be fixed with an update.
 - Erroneous: The information is erroneous and may indicate malicious intent.
 
-A seventh state, "fee paid", is for when a user has requested judgement and it is in progress. Information that is in this state or "erroneous" cannot be modified; it can only be removed by complete removal of the identity.
+A seventh state, "fee paid", is for when a user has requested judgement and it is in progress. Information that is in this state or "erroneous" is "sticky" and cannot be modified; it can only be removed by complete removal of the identity.
 
 Registrars gain trust by performing proper due diligence and would presumably be replaced for issuing faulty judgements.
 
+To be judged after submitting your identity information, go to the [Extrinsics UI](https://polkadot.js.org/apps/#/extrinsics) and select the `identity` pallet, then `requestJudgement`. For the `reg_index` put the index of the registrar you want to be judged by, and for the `max_fee` put the maximum you're willing to pay for these confirmations. Registrar 1's fee is 5 KSM, so that's what's in the image below.
+
+![Requesting judgement](/img/identity/08.jpg)
+
+This will make your identity go from unjudged:
+
+![An unjudged identity](/img/identity/07.jpg)
+
+To "waiting":
+
+![A pending identity](/img/identity/09.jpg)
+
+At this point, direct contact with the registrar is required. Each registrar will have their own set of procedures to verify your identity and values, and only once you've satisfied their requirements will the process continue.
```

Comment:
Should we add something which explains how to contact a registrar or find/select one in the first place?

Reply:
Done